public/js/safetymaps/modules/incidents/IncidentDetailsWindow.js

```diff
@@ -518,17 +545,34 @@ IncidentDetailsWindow.prototype.getIncidentKladblokHtml = function(format, incid
             }
             break;
         case "falck":
-            $.each(incident.Kladblokregels, function(i, k) {
-                kladblokHTML += "<tr><td>" + new moment(k.DTG).format("HH:mm") + "</td><td>" + me.linkify(dbkjs.util.htmlEncode(k.Inhoud)) + "</td></tr>";
-            });
+            if (this.editKladblokChat) {
+                kladblokHTML += this.kcDiv;
+            }
+            if (this.showKladblokChat) {
+                $.each(incident.Kladblokregels.sort(function (a, b) {
+                    var dateA = new moment(a.DTG);
+                    var dateB = new moment(b.DTG);
+                    return dateA._d - dateB._d;
+                }), function(i, k) {
+                    var style = k.IsChat ? "font-weight:normal !important; font-style:italic; !important" : "";
```

Comment:
Maybe nice to have a different style for chats sent by the same voertuignummer or user (different horizontal align just like chat apps).